src/node_util.cc

```diff
@@ -67,6 +69,20 @@ static void GetConstructorName(
   args.GetReturnValue().Set(name);
 }
 
+static void GetExternalValue(
+    const FunctionCallbackInfo<Value>& args) {
+  CHECK(args[0]->IsExternal());
+  auto isolate = args.GetIsolate();
```

Comment:
This is nice! Thank you!

This is not a blocker at all, but I'd prefer specify the type

```suggestion
  Isolate* isolate = args.GetIsolate();
```

Just a sugestion, not a blocker.